README.md

```diff
@@ -1,2 +1,3 @@
 # Food Hub
 April 2018 CA Cohort Mid Course Project
+Set up application structure
```

Comment:
Why is this added to the README?

Reply:
Tested to see if PR was set up properly. It has been removed.